### ISSUE TITLE
Reward can outnumber storage

### ISSUE THREAD

**Issue by twiernik** [DexterHuang/CyberCodeOnline#122]
One time using ai core I couldn't end the task. In response from the server there was a messsage saying something like: there are only 32 slots in inventory. That was even before the reward appeared. On mobile timer was increasing, in webapp timer was showing 0 all the time. This was done using .. workaround for number of cycles created in previous ticket.

**Comment by DexterHuang** [comment]
hmmm are you stuck now?@@

**Comment by twiernik** [comment]
I'm not stuck. I've decided to cancel task. I've started around a task that would take around 10hrs consuming around 40 ai core. After opening the game the time left showed 0 and in console I saw failed request that was supposed to end the task / receive revard and probably present to the user. I've tried several times but with same result. 
It is wild guess but I'm guessing when reward bucket / list exceeds 32 there is an exception.
